### ISSUE TITLE
Comments on ResourceQuota are incomplete

### ISSUE THREAD

**Issue by Huang-Wei** [kubernetes/kubernetes#93312]
The following comments are incomplete.

https://github.com/kubernetes/kubernetes/blob/98814409bab818d942e3c341d76eb594f12675a5/staging/src/k8s.io/api/core/v1/types.go#L5493-L5504

For example, comments for "ResourceQuotaScopeTerminating" should be:

```
// Match all pod objects where `spec.activeDeadlineSeconds >=0`.
```

Ref: https://kubernetes.io/docs/concepts/policy/resource-quotas/#quota-scopes

/good-first-issue

**Comment by Huang-Wei** [comment]
Per https://github.com/kubernetes/community/blob/master/sig-scheduling/charter.md

ResourceQuota is within scope of api-machinery.

/sig api-machinery
/remove-sig scheduling

**Comment by ironknight78** [comment]
Hi!! I'd like to help #investigate this problem @Huang-Wei, I'm a beginner, so I'll make my best. Can you please help me point me in the right direction?
What's the output today vs. how does the output should look like?

**Comment by ironknight78** [comment]
Oh I get it, the comments on top of the lines are not completely accurate on what the message displays, right? I think I got it

**Comment by Dean-Coakley** [comment]
@ironknight78 Yeah - But I've already opened a PR for this!  - See the linked PR `#93549`

So this is already done, just waiting on final merge.

**Comment by ironknight78** [comment]
Great Dean, thanks for that!
